style.css

```diff
@@ -0,0 +1,1079 @@
+}
+
+.about-paragraph-2 {
+  display: none;
+}
+
+.about-contact {
+  color: #646566;
+  font-size: 1.1rem;
+  font-weight: 500;
+  padding: 50px 50px 0 50px;
+}
+
+.about-email {
+  font-family: 'poppins', sans-serif;
+  color: #525255;
+  font-size: 1.1rem;
+  font-weight: 600;
+}
+
+/* about section ends here */
+
+/* section about-2 starts here */
+
+.about-2 {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.about-2 h2 {
+  margin: 50px 0 0 0;
+  width: 80%;
+  font-size: 1.8rem;
+}
+
+.about-hr {
+  background: #ff4f38;
+  margin: 20px 50% 20px 50%;
+  border: none;
+  width: 50px;
+  height: 3px;
+}
+
+.about-2-p-1 {
+  color: #737375;
+  font-size: 1.3rem;
+  letter-spacing: -1px;
+  padding: 10px 20px;
+  font-weight: 600;
+}
+
+.about-2-p-2 {
+  display: none;
+}
+
+.about-2 img {
+  width: 600px;
+  padding: 0 110px;
+  border: 2px solid #f0eeee;
+  margin: 0 0 100px 0;
+}
+
+.logo-hr {
+  background: #d7d8da;
+  border: none;
+  height: 1px;
+  width: 100vw;
+}
+
+.past-events {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 30px;
+}
+
+.past-events h2 {
+  margin: 50px 0 0 0;
+  font-size: 1.8rem;
+  width: auto;
+}
+
+.past-events p {
+  font-size: 1.3rem;
+  margin: 0 0 20px 0;
+}
+
+.cards-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.past-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 500px;
+  height: 280px;
+  margin: 0 0 40px 0;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  text-align: center;
+}
+
+.past-card h1 {
+  font-size: 3rem;
+  font-weight: 700;
+}
+
+.past-card p {
+  color: #fff;
+  font-weight: 500;
+}
+
+.past-image-1 {
+  background: url(./assests/image-1.png);
+  background-size: contain;
+}
+
+.past-image-2 {
+  background: url(./assests/image-2.png);
+  background-size: contain;
+}
+
+/* section about-2 ends here */
+
+/* partners section starts here */
+
+.partner,
+.partner-2 {
+  background-color: #404041;
+  padding: 20px;
+  color: #fff;
+  text-align: center;
+  font-size: 1.2rem;
+}
+
+.partner h1,
+.partner-2 h1 {
+  margin: 30px 0 0 50px;
+  font-weight: 400;
+  text-align: center;
+}
+
+.partner-images,
+.about-images {
+  margin: 20px 0 20px 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.partner-images img {
+  padding: 20px;
+  width: 180px;
+}
+
+.partner-images :nth-child(2) {
+  width: 200px;
+}
+
+.partner-images :nth-child(4) {
+  width: 210px;
+}
+
+.partner-images :nth-child(5) {
+  width: 240px;
+}
+
+/* partners section ends here */
+
+/* footer starts here */
+
+footer {
+  margin: 50px 0  50px 0;
+  padding: 10px 50px;
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 100%;
+}
```

Comment:
- [OPTIONAL] In the _**mobile**_ version of the _**Home**_ page, the elements in the `footer` are overlapping. I would advise you to space them a bit. 😎

![image](https://github.com/Kaiserabbas/Capstone/assets/77106962/4ae0f585-2087-4d97-9e54-02c1c03fadc4)

#### Hint: I would advise you to reduce the `padding` value on the x-axis that you declared in the `footer` selector from `50px` to `20px`. 💻 🖱️